app/Http/Requests/Nordigen/ConnectNordigenBankIntegrationRequest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2023. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+namespace App\Http\Requests\Nordigen;
+
+use App\Http\Requests\Request;
+use App\Libraries\MultiDB;
+use App\Models\Company;
+use App\Models\User;
+use Cache;
+use Log;
+
+class ConnectNordigenBankIntegrationRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'lang' => 'string',
+            'institution_id' => 'string',
+            'redirect' => 'string', // TODO: @turbo124 @todo validate, that this is a url without / at the end
+        ];
+    }
+
+    // @turbo124 @todo please check for validity, when request from frontend
+    public function prepareForValidation()
+    {
+        $input = $this->all();
+
+        if (!array_key_exists('redirect', $input)) {
```

Comment:
As above, the one time token is our security layer here.

Reply:
i did not meant this property for security, it was just how we can validate the provided input.
I will close this conversation, because we can discuss this on the others